posts/2022/05/thoughts-on-reviewing-for-accessibility.md

```diff
@@ -0,0 +1,185 @@
+<!--
+.. title: Thoughts on reviewing for accessibility: Maybe there is a checklist?
+.. slug: thoughts-on-reviewing-for-accessibility
+.. date: 2022-05-25 08:00:00 UTC-00:00
+.. author: Isabela Presedo-Floyd
+.. tags: JupyterLab, Accessibility, JLabA11y
+.. category: JLabA11y
+.. link:
+.. description:
+.. type: text
+-->
+
+![Checkmark and x-filled checkboxes in a repeating pattern.](/images/jlabaccess4.png)
+
+## JupyterLab Accessibility Journey Part 4
+
+Remember how [my last post in this series](https://labs.quansight.org/blog/2021/09/not-a-checklist/) 
+called out accessibility as much more complex than a checklist? True to my 
+sense of humor, this blog post is now a checklist. Irony? I don’t know the 
+meaning of the word.
+
+Okay, okay. But seriously, when I’m not making my own work, much of my time is 
+spent reviewing other people’s work. Whether it’s design files, code 
+contributions, blog posts, documentation, or who-knows-what-this-week, I often 
+find myself asking questions and giving feedback about accessibility in the 
+review process. This has prompted multiple people to ask me what it is I’m 
+considering when I review for accessibility. Enough people have now asked that 
+I’ve decided to write something down.
+
+<!-- TEASER_END -->
+
+I’d like to remind readers everywhere that other people have written similar 
```

Comment:
I'd have to look for examples that I think are scoped enough to be relevant, but I can do that. I don't have some immediately in mind at the moment (most are just reiterating WCAG or some other laws, which I'm not sure if that's a good scope).

Reply:
I don't think the blog post needs such links. But thought it could be an opportunity to give a nod, as acknowledgment, to some of the people you look up to in this space